supervisor/api/__init__.py

```diff
@@ -107,6 +118,17 @@ async def load(self) -> None:
         self._register_store()
         self._register_supervisor()
 
+        if static_resource_configs:
+
+            def process_configs() -> list[web.StaticResource]:
+                return [
+                    web.StaticResource(config.prefix, config.path)
+                    for config in static_resource_configs
+                ]
+
+            for resource in await self.sys_run_in_executor(process_configs):
+                self.webapp.router.register_resource(resource)
+
```

Comment:
It would be a bit nicer to group the executor jobs into a single call like we do in core since the cost of creating the future and waiting for it etc is not small

Reply:
Isn't that what I did here? I'm making all the static resources in one executor call and then registering all those afterwards

Reply:
Ignore me. I misread the code.